gwpy/signal/window.py

```diff
@@ -83,7 +85,8 @@ def canonical_name(name):
     'hann': .5,
     'hamming': .5,
     'nuttall': .656,
-    'triang': .5
+    'triang': .5,
+    'planck': .01
```

Comment:
for my edification, can you explain the choice of `.01` here?

Reply:
It's definitely somewhat arbitrary. By default, the `TimeSeries.taper()` method which wraps around `signal.window.planck()` automatically tapers to the second stationary point from either side of the signal, so its taper length can vary wildly. To come up with `.01`, I chose the duration of one cycle of a sinusoid at 100 Hz, which is roughly on the same order as what we would commonly taper out of our waveforms.

Reply:
if the recommended overlap for this window is not actually known, I would rather omit it from this mapping, so that the user needs to manually select an overlap.

Reply:
I'm happy enough with that setup. That has been done in commit f58afa24937a87d82ab7a664b488f2807c97a51f